jenkins/internal-ci/generic/pr/component_test/Motr_Pre_merge_DTM_Integration.groovy

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env groovy
+// CLEANUP REQUIRED
```

Comment:
Can we remove this comment if not required

Reply:
Done